cvat-core/src/api-implementation.js

```diff
@@ -222,30 +216,26 @@
                 owner: isString,
                 search: isString,
                 status: isEnum.bind(TaskStatus),
+                without_tasks: isBoolean,
```

Comment:
Why snake case here?

Reply:
Due to we rely on fields in server requests params. Added replacing.